<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1227</width>
    <height>912</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>900</width>
      <height>700</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>30</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Треугольник</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Круг</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ромб</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Квадрат</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Прямоугольник</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Шестиугольник</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Звезда</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Эллипс</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>730</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить всё</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>↶</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>↷</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>-10</y>
      <width>16</width>
      <height>881</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>730</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Площадь в пикселях:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>760</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Периметр в пикселях:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>760</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>790</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_vverh">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_vlevo">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_vpravo">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_vniz">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>380</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>380</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_koords">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>460</y>
      <width>170</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Изменение координат</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_drawRomb">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>540</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Chandas</family>
      <pointsize>26</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>◇</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_drawKvadrat">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>540</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chandas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>□</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_drawTreugolnik">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>540</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chandas</family>
      <pointsize>23</pointsize>
     </font>
    </property>
    <property name="text">
     <string>△</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_drawKrug">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>540</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chandas</family>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>◯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_drawRect">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>590</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chandas</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>▭</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_drawEllips">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>590</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chandas</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>⬭</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_drawHex">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>590</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FreeSans</family>
      <pointsize>23</pointsize>
     </font>
    </property>
    <property name="text">
     <string>⬡</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_drawZvezda">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>590</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FreeSans</family>
      <pointsize>28</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>☆</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1227</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPaint_2_0">
    <property name="title">
     <string>Paint 2.0</string>
    </property>
   </widget>
   <addaction name="menuPaint_2_0"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
